satellite/console/projectpaymentinfos.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package console
+
+import (
+	"context"
+	"time"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+)
+
+// ProjectPaymentInfos is project payment infos store interface
+type ProjectPaymentInfos interface {
+	Create(ctx context.Context, info ProjectPaymentInfo) (*ProjectPaymentInfo, error)
+	GetByProjectID(ctx context.Context, projectID uuid.UUID) (*ProjectPaymentInfo, error)
+	GetByPayerID(ctx context.Context, payerID uuid.UUID) (*ProjectPaymentInfo, error)
+}
+
+// ProjectPaymentInfo contains project payment info
+type ProjectPaymentInfo struct {
```

Comment:
Any reason this is not called `ProjectPayment`?

Reply:
Because it is not the actual payment method or so, only a reference to the actual struct stored on stripe or any other payment service

Reply:
Yeah, it's not wrong. But I'm not sure whether we gain here from adding the `Info` at the end. It's something that would be sufficient to cover in documentation rather than type name.

Reply:
okay